tee-worker/litentry/core/stf-task/receiver/src/handler/assertion.rs

```diff
@@ -276,11 +269,17 @@ where
 		Assertion::NftHolder(nft_type) =>
 			lc_assertion_build_v2::nft_holder::build(req, nft_type, &context.data_provider_config),
 
-		Assertion::Dynamic(smart_contract_id) => lc_assertion_build::dynamic::build(
-			req,
-			smart_contract_id,
-			context.assertion_repository.clone(),
-		),
+		Assertion::Dynamic(smart_contract_id) => {
+			#[cfg(test)]
+			{
+				std::thread::sleep(core::time::Duration::from_secs(5));
```

Comment:
for what reason do we need the sleep? Is it awaits something to be created before?

Reply:
It's used by test to detect if two assertions are processed in parralel.